test/util_test.go

```diff
@@ -30,10 +32,16 @@ var (
 // newKine will panic in case of error
 //
 // newKine will return a context as well as a configured etcd client for the kine instance
```

Comment:
While we are here, can you add a note about what `qs` is (and update the rest of the godoc, as it is slightly outdated)?

Reply:
Done